index.js

```diff
@@ -1,3 +1,4 @@
+console.clear();
```

Comment:
As we use this only during development, please delete it before creating a PR. 🧹 
```suggestion

```